PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -35,6 +36,14 @@ class AccountViewModel: ObservableObject {
         self.userDefaults = userDefaults
         self.notificationCenter = notificationCenter
         self.premiumUpgradeViewModelFactory = premiumUpgradeViewModelFactory
+        self.isPremium = user.status == .premium
+
+        userStatusListener = user
```

Comment:
curious to see why this method vs. `.store()`

Reply:
Hey @cyndichin , I'm not sure I follow here: can you please clarify?

Reply:
I just noticed in our code base we use the following to store our cancellable objects, so was curious to see why this method was preferred over using `.store`

<img width="471" alt="image" src="https://user-images.githubusercontent.com/6743397/221633895-04d464e0-c7be-4256-a896-3ba58f584660.png">

Reply:
Oh I see now, thanks for clarifying! Basically, `,store(in: )`: the reason I did not use that is because, being this a single subscription, we don't really need a `Set` for it, but I can totally change it if you feel that makes the code more clear or more consistent.

Reply:
either works, was curious about the difference!